client/me/account/main.jsx

```diff
@@ -399,7 +398,7 @@ const Account = createReactClass( {
 
 	renderJoinDate() {
 		const { translate, moment } = this.props;
-		const dateMoment = moment( user.get().date );
+		const dateMoment = moment( user().get().date );
```

Comment:
In this component, we should be retrieving user info from Redux. It's already `connect`ed.

Reply:
Done!